docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md

```diff
@@ -16,7 +16,7 @@ If you want to see the final product of this tutorial before you plunge in, chec
 
 ## Improving accessibility
 
-First, we make our plugin accessible for users who rely on keyboards for navigation. We want to ensure that pressing <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd> will move focus around in the form view, and pressing `esc` will close it.
+First, we make our plugin accessible for users who rely on keyboards for navigation. We want to ensure that pressing <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd> will move focus around in the form view, and pressing <kbd>Esc</kbd> will close it.
```

Comment:
Are these keys behaving like this after recent changes?

Reply:
From my testing - everything works except the escape key. But I fixed it in the tutorial files.